A problem occured during compiling

```
root@kali:~/demovfuscator# make
clang++ -Wall -Wextra -pedantic -std=c++11 -g -c demov.cpp
In file included from demov.cpp:7:
In file included from ./demov.hpp:14:
In file included from ./dishlp.hpp:14:
./stackMachine.hpp:61:7: error: no matching function for call to 'Z3_get_numeral_uint64'
                if (Z3_get_numeral_uint64(ctx, cn,(unsigned long long*)  &ret)) {
                    ^~~~~~~~~~~~~~~~~~~~~
/usr/include/z3_api.h:4563:17: note: candidate function not viable: no known conversion from 'unsigned long long *' to 'uint64_t *'
      (aka 'unsigned long *') for 3rd argument
    bool Z3_API Z3_get_numeral_uint64(Z3_context c, Z3_ast v, uint64_t* u);
                ^
In file included from demov.cpp:7:
In file included from ./demov.hpp:14:
In file included from ./dishlp.hpp:14:
./stackMachine.hpp:78:14: error: call to member function 'bv_val' is ambiguous
                ret = ctx->bv_val((unsigned long long) cur.imm, 32);
                      ~~~~~^~~~~~
/usr/include/z3++.h:2931:26: note: candidate function
    inline expr context::bv_val(int n, unsigned sz) { sort s = bv_sort(sz); Z3_ast r = Z3_mk_int(m_ctx, n, s); check_error(); return expr(*this, r); }
                         ^
/usr/include/z3++.h:2932:26: note: candidate function
    inline expr context::bv_val(unsigned n, unsigned sz) { sort s = bv_sort(sz); Z3_ast r = Z3_mk_unsigned_int(m_ctx, n, s); check_error(); re...
                         ^
/usr/include/z3++.h:2933:26: note: candidate function
    inline expr context::bv_val(int64_t n, unsigned sz) { sort s = bv_sort(sz); Z3_ast r = Z3_mk_int64(m_ctx, n, s); check_error(); return exp...
                         ^
/usr/include/z3++.h:2934:26: note: candidate function
    inline expr context::bv_val(uint64_t n, unsigned sz) { sort s = bv_sort(sz); Z3_ast r = Z3_mk_unsigned_int64(m_ctx, n, s); check_error(); ...
                         ^
/usr/include/z3++.h:2935:26: note: candidate function not viable: no known conversion from 'unsigned long long' to 'const char *' for 1st argument
    inline expr context::bv_val(char const * n, unsigned sz) { sort s = bv_sort(sz); Z3_ast r = Z3_mk_numeral(m_ctx, n, s); check_error(); ret...
                         ^
/usr/include/z3++.h:2936:26: note: candidate function not viable: no known conversion from 'int' to 'const bool *' for 2nd argument
    inline expr context::bv_val(unsigned n, bool const* bits) {
                         ^
2 errors generated.
```

I installed all the dependencies(via github with latest versions).

--- a/stackMachine.hpp
+++ b/stackMachine.hpp
@@ -58,7 +58,7 @@ uint64_t get_label(std::stack<element, c> st, uint64_t *addr) {
 		z3::func_decl v = m[i];
 		assert(v.is_const());
 		z3::expr cn = m.get_const_interp(v);
-		if (Z3_get_numeral_uint64(ctx, cn,(unsigned long long*)  &ret)) {
+		if (Z3_get_numeral_uint64(ctx, cn,(uint64_t*)  &ret)) {
 			*addr = mv;
 			return ret;
 		}
@@ -75,7 +75,7 @@ z3::expr gen_expr(std::stack<element, c> *st,
 	st->pop();
 
 	if (cur.type == ELE_CONST) {
-		ret = ctx->bv_val((unsigned long long) cur.imm, 32);
+		ret = ctx->bv_val((uint64_t) cur.imm, 32);
 	} if (cur.type == ELE_MEM) {
 		if (!*mem_valid) {
 			*mv = (uint64_t) cur.mem.disp;

